python/cudf/cudf/tests/test_orc.py

```diff
@@ -180,14 +180,14 @@ def test_orc_read_statistics(datadir):
         (
             file_statistics,
             stripes_statistics,
-        ) = cudf.io.orc.read_orc_statistics(path)
+        ) = cudf.io.orc.read_orc_statistics([path])
```

Comment:
```suggestion
        ) = cudf.io.orc.read_orc_statistics([path, path])
```

Just to try multiple path option.